src/ConfigFactory.Avalonia.Demo/Models/DemoConfig.cs

```diff
@@ -78,4 +81,35 @@ public static ObservableCollection<string> GetThings()
             return value is true;
         });
     }
+
+    /// <summary>
+    /// Example implementation of i18n using the .resx Embedded Resource
+    /// </summary>
+    public override string Translate(string input)
+    {
+        string result = input;
+        PropertyInfo? prop = null;
+
+        if (!string.IsNullOrWhiteSpace(input))
+        {
+            // get resource property
+            prop = (typeof(Translations))
+                .GetProperty(input, BindingFlags.Public | BindingFlags.Static | BindingFlags.NonPublic);
+        }
+
+        if (prop != null)
+        {
+            // use property getter
+            var getterInfo = prop.GetGetMethod(nonPublic: true);
+            if (getterInfo != null)
+            {
+                if (Delegate.CreateDelegate(typeof(Func<string>), getterInfo) is Func<string> fn)
+                {
+                    result = fn();
+                }
+            }
+        }
```

Comment:
I think this can be much more concise

```suggestion
        MethodInfo? getterInfo = prop?.GetGetMethod(nonPublic: true);
        if (getterInfo?.Invoke(null, null) is Func<string> fn) {
            result = fn();
        }
```

Reply:
It is more about personall code style, but I don't mind following your suggestion.